### DISCUSSION TITLE
Heya, just FYI, we've moved the GitHub Organization!

### DISCUSSION THREAD

**Discussion by 6mile** [securestack-octernship/SecureStack-Octernship#28] [Announcements]
We moved the GitHub Organization that this Classroom is using to securestack-octernship because there were so many people in the org.  That's a good thing!  

As part of this move we migrated all of your forked repositories to this new GitHub org.  

You don't have to do anything, we're just letting you know.  Thanks for being a part of the fun!  Cheers!

**Comment by mukulpadwal** [comment]
@6mile How will the results be announced?

**Comment by mukulpadwal** [comment]
Hi @6mile any updates regrading results?
